aspnetcore/fundamentals/host/index.md

```diff
@@ -15,3 +15,9 @@ uid: fundamentals/host/index
 * [Generic Host](xref:fundamentals/host/generic-host) (ASP.NET Core 2.1 or later) &ndash; Suitable for hosting non-web apps (for example, apps that run background tasks). In a future release, the Generic Host will be suitable for hosting any kind of app, including web apps. The Generic Host will eventually replace the Web Host.
 
 For hosting ASP.NET Core *web apps*, developers should use the Web Host based on [IWebHostBuilder](/dotnet/api/microsoft.aspnetcore.hosting.iwebhostbuilder). For hosting *non-web apps*, developers should use the Generic Host based on [HostBuilder](/dotnet/api/microsoft.extensions.hosting.hostbuilder).
+
+<xref:fundamentals/host/hosted-services>  
+Learn how to implement background tasks with hosted services in ASP.NET Core.
+
+<xref:fundamentals/configuration/platform-specific-configuration>  
+Discover how to enhance an ASP.NET Core app from a referenced  or unreferenced assembly using an `IHostingStartup` implementation.
```

Comment:
- Remove the extra space after "referenced".
- Link to the IHostingStartup API ref